README.md

```diff
@@ -1 +1,7 @@
 # hw01
```

Comment:
Your reflection of your process was very thorough and it seems like you learned a lot in the process of completing this assignment. It was very clear what you did, what issues you had, and how you solved them.